### PR TITLE
Speed up application tests

### PR DESCRIPTION
**Pull request by taehoonlee**
Compared to the builds of previous versions (e.g., [2.1.1](https://travis-ci.org/keras-team/keras/builds/302166968) and [2.1.0](https://travis-ci.org/keras-team/keras/builds/301611719)), test times increased by about 10 min nowadays for all pytest environments.

- TF: 16-18 min -> 29-42 min
- TH: 23-25 min -> 26-36 min
- C: 13-17 min -> 19-29 min

This is because new applications, NASNet and DenseNet, have been added. Unlike other applications (resnet, inception, vgg, ...), the new applications share internal base functions with different parameters (NASNetMobile, NASNetLarge call NASNet; DenseNet121, DenseNet169, DenseNet201 call DenseNet). Thus, we need not to test all the variants, but to test a single representative application. This PR addresses this issue by introducing random test numbers.

### CODE REVIEW

**Comment by taehoonlee** [comment]
The results are:

- TF: 26-29 min
- TH: 23-24 min
- C: 16-20 min

File: tests/keras/applications/applications_test.py [hunk 1]
```diff
@@ -286,51 +296,43 @@ def test_mobilenet_image_size():
...
 @keras_test
-@pytest.mark.parametrize('fun', [
-    applications.DenseNet121,
-    applications.DenseNet169,
-    applications.DenseNet201],
-    ids=['DenseNet121', 'DenseNet169', 'DenseNet201'])
-def test_densenet(fun):
+def test_densenet():
+    random.seed(time.time())
+    fun, _ = DENSENET_LIST[random.randint(0, 2)]
```
Comment (fchollet) [inline tests/keras/applications/applications_test.py]: You can use `random.choice` instead

Comment (taehoonlee) [reply tests/keras/applications/applications_test.py]: Updated, @fchollet.

**Comment by taehoonlee** [comment]
@fchollet This PR can also be merged as well as #9059.

**Pull request merged**